mobile-app/app/screens/AppNavigator/screens/Dex/hook/SwappableTokens.ts

```diff
@@ -58,6 +58,7 @@ export function useSwappableTokens (fromTokenId: string | undefined): TokenPrice
           available: new BigNumber(ownedToken === undefined ? 0 : ownedToken.amount),
           token: {
             displaySymbol: token.displaySymbol,
+            id: token.id,
```

Comment:
Is this used? There's also a duplicate value of this in `tokenId` prop in the same object